pipeline/07-export.R

```diff
@@ -271,40 +271,71 @@ assessment_card <- dbGetQuery(
 #- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 message("Preparing data for Desk Review export")
 
-# Get the number of apartments for class 211 and 212 properties. Since
-# char_apts is recorded on the card level, we need to aggregate them to
-# the PIN level
-num_apts_by_pin <- assessment_card %>%
-  filter(char_class == "211" | char_class == "212") %>%
-  select(meta_pin, char_apts) %>%
-  # Convert the long format for char_apts to a shorter one that's easier
+format_char_apts <- function(char_apts) {
```

Comment:
Since multiple sheets are using this logic now, we extract it out into a dedicated helper function.